src/blueberry-muffins.md

```diff
@@ -0,0 +1,54 @@
+# Blueberry Muffins
+
+![blueberry-muffins](/data/pix/blueberry-muffins.webp)
+
+- ⏲️ Prep time: 20 min
+- 🍳 Baking time: 20 min
+- 🍽️ Servings: 12
+
+## Equipment
+
+-At least 12 muffin tins / A cake tin and optionally a hand mixer
+
+## Ingredients
+
+- 250g / 9 oz Blueberries 
+- 125g / 4½ oz Butter*
+- 150g / 5½ oz Sugar (Of course  you can lower the amount to your liking)
+- 1 Vanilla Pod or Teaspoon of vanilla sugar
+- 2 Eggs
+- 125ml / 4.22 fluid ounces of Milk
+- 300g /  11 oz Flour
+- 2 Teaspoons of Backing Powder
+- 125g / 4½ oz Powdered sugar (optional)
+- Half a lemon (optional)
+*If you don't have Butter, you can substitute it with 100g of Oil / 3½ oz of Oil suited for baking. Notice the measurement in grams / ounces and not milliliter and fluid ounces
+
+## Directions
+
+1. If using a vanilla pod, wash it with water, extract it's vanilla and add it to the molten butter. Put the now empty vanilla pod into the milk. This way every single bit of the vanilla get's extracted
+2. Wash the blueberries
+2. For the liquid ingredients, melt the butter, do not cook it
```

Comment:
You have 2 `2.`s in a row. I don't care if you fix it, but thought I'd point it out in case you cared. Markdown fixes numbering for you anyways so whether or not the numbers are correct makes no difference in the end.

Reply:
Thank you for pointing that out! I'll fix it!
And thank you so much for all the other suggestions and looking through the file in general.
I'll make sure to step up my english-game for future recipes

Reply:
np. I assume your portfolio page is also in development? And just so you know, the 2 links to your portfolio on your website don't point to where you think they do.

one of them points to `https://alexplatonov.com/alexplatonov.com/portfolio` and the other points to `https://alexplatonov.com/halexplatonov.com/portfolio`.

The 2 anchor tags you have are these:

```html
<a href="alexplatonov.com/portfolio">
<a href="halexplatonov.com/portfolio">
```

If you don't have a protocol specified, HTML assumes these are relative paths on your site, not URLs

You can change them to any one of these and they will work:

```html
<a href="portfolio">
<a href="/portfolio">
<a href="https://alexplatonov.com/portfolio">
```

assuming 'portfolio' ends up being a real route to something.

Reply:
Okay, the numbering is now fixed, thank's for pointing that out!
Oh! You are right! I have not even noticed that i put in broken links. I now gave them a proper linking that indicate a work in progress! :)
It's always good to have another pair of eyes watching over the code, thank you!